content/tutorials/interoperability/get-started-evm-sidechain.md

```diff
@@ -61,12 +62,12 @@ To connect a MetaMask wallet to the bridge, go to the [EVM Sidechain bridge](htt
 
 ![Connect MetaMask Wallet](img/evm-sidechain-connect-metamask.png "Connect MetaMask wallet")
 
-### 2. Initiate the Transfer of Funds 
+### 2. Initiate the Transaction 
 
 Now that both Xumm and MetaMask wallets are connected to the bridge, you can select the direction of the bridge, the amount to send, and the destination address.
 
 - **Direction of the bridge**: This is the direction of transfer; it can be either EVM sidechain → XRP Ledger Devnet or XRP Ledger Devnet → EVM sidechain. Use the “Switch Network” button to switch the direction of transfer.
-- **Amount to send**: This is the amount that you want to transfer to the other side. Note that there is a fee to use the bridge.
+- **Amount to send**: This is the amount that you want to send to the other side. Note that there is a fee to use the bridge.
     - Network fees: The fees required by the network for your transactions.
     - Commission: The bridge applies a commission for every transaction completed. This is to prevent spam and distributed denial of service attacks (DDOS).
 - **Destination address**: The address on the destination chain where you want to receive funds.
```

Comment:
From:
- **Destination address**: The address on the destination chain where you want to receive funds.
To:
- **Address**: The address on the chain where you want the funds.